F/FLAC/build_tarballs.jl

```diff
@@ -28,7 +27,7 @@ install_license COPYING.Xiph
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = expand_cxxstring_abis(supported_platforms())
+platforms = supported_platforms()
```

Comment:
????

Reply:
Reverted. I was just trying things to see if I could get it to compile on the other platforms.